kolibri/plugins/example_plugin/kolibri_plugin.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+"""
+Kolibri example plugin
+======================
+
+Copy and modify this code for your own plugin.
+
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+from django.utils.translation import ugettext as _
+
+from kolibri.plugins.base import KolibriPluginBase
+from kolibri.plugins.hooks import NAVIGATION_POPULATE
+
+
+class ExamplePlugin(KolibriPluginBase):
+
+    @classmethod
+    def enable(cls, config):
```

Comment:
I'm going with a new suggested property `auto_add_to_installed_apps` for convenience so that the below can be skipped since probably all plugins will be django apps that add themselves to INSTALLED_APPS.

I think that instantiation of the plugin should only happen within a Django runtime env, thus I left enable/disable as static methods. No database connection required, no HTTP server, none of all that... the plugin is free to do what it wants, and if it wants to initialize the django stack (`django.setup()`), it will have full responsibility of doing so :)